cadasta/utilities/resources.py

```diff
@@ -79,6 +80,23 @@ def resource_url(path):
     return str(url.toString())
 
 
+def is_valid_url(url):
```

Comment:
Do you have some unit test for that ?, maybe you could http://doc.qt.io/qt-4.8/qurl.html#isValid